CONTRIBUTING.md

```diff
@@ -124,6 +132,12 @@ development with the following:
   end
 ```
 
+## Granting Yourself Admin Access
+
+Once you've created an account with MyUSA, open the Rails console and change your admin status to "true". This will allow you to create and schedule messages.
+
+Note: scheduled messages may not send if you're using over the weekend, which is due to the business_time gem referenced above.
```

Comment:
I'd say this deserves to be highlighted one way or another, because you can't do much without being an admin.  Not sure if there's a better way of getting admin access or if the way we did it (as described here) is appropriate.